### PR TITLE
LPD-29276 - Change the definition of the RolesGroup interface by replacing LabelValueObject with LabelNameObject

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-29276

### CODE REVIEW

**Comment by rebsilva** [comment]
just started reviewing! :cherry_blossom:

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/tests/RoleUtils.spec.ts [hunk 1]
```diff
@@ -137,3 +139,80 @@ it('Assert roles in User Notification', () => {
 		},
 	]);
 });
+
+it('verify that handleMultiSelectRoleItemsChange generates new recipients in the correct data structure', () => {
```
Comment (rebsilva) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/tests/RoleUtils.spec.ts:143]: everything looks great, you just forgot to rename the `RoleUtils.spec.ts` to `RolesUtil.spec.ts` following your previous changes

**Review by rebsilva — CHANGES_REQUESTED** [review]
hi @danvb112, great work on discovering the cause of this bug! :tada: just made a comment, let me know what you think about it!

**Comment by danvb112** [comment]
ci:test:sf

**Review by rebsilva — APPROVED** [review]
LGTM! great work @danvb112! :tada: :balloon:

**Comment by danvb112** [comment]
ci:test:playwright

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:notification

**Comment by danvb112** [comment]
ci:test:object

**Comment by danvb112** [comment]
CI failures are not related!

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**